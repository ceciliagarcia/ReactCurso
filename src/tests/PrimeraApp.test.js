import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import React from 'react';

import PrimeraApp from '../PrimeraApp';

describe('Prueba en <PrimeraApp />', () => {

   // test('debe de mostrar el mensaje "Hola, Cecilia"', () => {
        
       // const saludo = "Hola, soy Cecilia";

       // const { getByText } = render(<PrimeraApp saludo={saludo}/>);

       // expect(getByText(saludo)).toBeInTheDocument();


   // })

   test('debe de mostrar <PrimeraApp /> correctamente', () => {

        const saludo = 'Hola, soy Cecilia';
        const wrapper = shallow(<PrimeraApp saludo={saludo} />);

        expect(wrapper).toMatchSnapshot();
   })
   
    

});