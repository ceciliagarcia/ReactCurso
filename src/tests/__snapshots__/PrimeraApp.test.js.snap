// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en <PrimeraApp /> debe de mostrar <PrimeraApp /> correctamente 1`] = `
<Fragment>
  <h1>
     
    Hola, soy Cecilia
     
  </h1>
  <p>
     
  </p>
</Fragment>
`;
